static/app/views/starfish/utils/getModuleFromLocation.tsx

```diff
@@ -0,0 +1,19 @@
+import {Location} from 'history';
+
+import {t} from 'sentry/locale';
+
+export const MODULES = {
+  api: t('API Module'),
+  database: t('DB Module'),
+  'endpoint-overview': t('Endpoint Overview'),
+};
+
+export function getModuleFromLocation(location: Location) {
```

Comment:
Same here, it doesn't just return modules, it's like a reverse router. I think it can be called something like `extractRoute` or something similar, since you use it to construct URLs?